apps/win/mintty/mintty.talon

```diff
@@ -1,17 +0,0 @@
-app: mintty
--
-tag(): terminal
-tag(): user.file_manager
-tag(): user.generic_terminal
-tag(): user.git

```

Comment:
i'm not sure if this added somewhere else, but this tag assertions are missing from mintty.py. i'm not sure if this is a side effect of once the actions being gone the original file is removed?  i'm not sure if this is affecting either apps as well. the guake.py file seemed to have them added as `apps/platform/linux/guake.py
7:ctx.tags = ['user.git', 'user.kubectl', 'user.tabs', 'terminal']`, so maybe something did mess up with the script here?

Reply:
On my machine, talon created mintty_actions.py. Looks like @knausj85 merged that file manually to mintty.py and didn't copy the tags.

Reply:
Good catch, looks like I only missed mintty